cogs/help.py

```diff
@@ -259,6 +296,11 @@ async def on_thread_member_remove(self, member: ThreadMember):
             return
 
         await close_help_thread("EVENT", thread, thread_author)
+        await stats_client.update_thread(
```

Comment:
Why is this only in the member remove/leave event? Shouldn't it be in the close thread func instead?

Reply:
It requires too much context to be moved there without extra refactoring, although it would be ideal.